### ISSUE TITLE
Blur effect does not apply to context menu and gnome shell

### ISSUE THREAD

**Issue by fsevenm** [yilozt/mutter-rounded#111]
I managed to install the package and got the expected blur effect, but I wonder if there's a way to apply the blur effect to the context menu. Furthermore, gnome shell and its extensions are also not affected with this blur effect. I am not sure if this is the limitation of this patch.
![image](https://user-images.githubusercontent.com/29963765/232028357-073b70e2-e418-41dc-8abf-fd2298bed53a.png)

**Comment by assiless** [comment]
is this vscode/electron/chromium app?

**Comment by fsevenm** [comment]
Correct, it's vscode. But even on Firefox and everywhere else, the menu and context menu all not blurred.

**Comment by renankogan** [comment]
> Correct, it's vscode. But even on Firefox and everywhere else, the menu and context menu all not blurred.

For this to work in Firefox (_I'm not saying it will work, but I'm saying that it would work in case you managed to do this in normal GTK apps_), you would need to set your context menu to **native** in about:config, `widget.gtk.native-context-menus` to `true`
